tests/set1.py

```diff
@@ -12,3 +12,7 @@ def test_Set():
 
     s: str
     s = b.pop()
+
+s1: set[str] = {"a", "b", "c", "a"}
+s2: set[i32] = {1, 2, 3, 1}
+s3: set[tuple[i32, i32]] = {(1, 2), (2, 3), (4, 5)}
```

Comment:
We should be adding an integration test for this, not a reference test. A reference test is only needed when you want to check/verify the intermediate output or when adding an integration test is not possible/feasible.

Since in this PR you implemented support for global variables of type `set`, we want to know/check if an example with a global `set` variable compiles and runs and produces expected output.

Reply:
Also, I think we should be adding a new test for this instead of updating an old test. Old test should be updated only when there is close similarity between the old test and the new test. We should mostly be adding new tests since we would like to keep integration tests short, testing only one specific functionality in a single test.

Reply:
I am very sorry for this! I overlooked the folder name. I had wanted to add this to `integration_tests` only. I am pushing the changes in a while